### PR TITLE
Add initial ansible execution and use it for network config

### PR DESCRIPTION
**Pull request by slagle**
Adds the initial handling of ansible execution, within
pkg/util/ansible_execution.go, which will create CR's of AnsibleEE.

The network configuration illustrates how the ansible execution is used.
The deployment package will contain other files that trigger ansible
executions for different stages of the deployment.

Signed-off-by: James Slagle <jslagle@redhat.com>

### CODE REVIEW

**Comment by slagle** [comment]
This includes the revert at https://github.com/openstack-k8s-operators/dataplane-operator/pull/22 just so I could test this out for the time being. I will rebase it once that PR is merged.

File: config/crd/bases/dataplane.openstack.org_openstackdataplanenodes.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
+    controller-gen.kubebuilder.io/version: v0.9.2
```
Comment (fao89) [inline config/crd/bases/dataplane.openstack.org_openstackdataplanenodes.yaml:6]: it is reverting the kubebuilder version, please remove your `./bin/controller-gen` so you can get an updated one

File: controllers/openstackdataplanenode_controller.go [hunk 2]
```diff
@@ -60,7 +62,7 @@ type OpenStackDataPlaneNodeReconciler struct {
 // For more details, check Reconcile and its Result here:
```
Comment (fao89) [inline controllers/openstackdataplanenode_controller.go:62]: we need to add (line 50)
```
//+kubebuilder:rbac:groups=ansibleee.openstack.org,resources=openstackansibleees,verbs=get;list;watch;create;update;patch;delete
```

File: config/crd/bases/dataplane.openstack.org_openstackdataplanenodes.yaml (hunk 1 above)
Comment (slagle) [reply config/crd/bases/dataplane.openstack.org_openstackdataplanenodes.yaml:6]: Done.

File: controllers/openstackdataplanenode_controller.go (hunk 2 above)
Comment (slagle) [reply controllers/openstackdataplanenode_controller.go:61]: Done.

**Review by fao89 — APPROVED** [review]

**Pull request merged**